sequencer.h

```diff
@@ -116,3 +116,7 @@ int prepare_branch_to_be_rebased(struct replay_opts *opts, const char *commit);
 void print_commit_summary(const char *prefix, const struct object_id *oid,
 			  unsigned int flags);
 #endif
+
+void parse_strategy_opts(struct replay_opts *opts, char *raw_opts);
+int write_basic_state(struct replay_opts *opts, const char *head_name,
+		      const char *onto, const char *orig_head);
```

Comment:
I wonder if write_basic_state() really needs to be public in sequencer.{h,c}. Do you have plan to reuse it?

Reply:
No, I did it to have access to ``rebase_path_*()``.

Reply:
Ok, then.

Reply:
Perhaps it would be better to make those we need public, like I did with the todo-list file?

Reply:
Yeah, that's another possible solution. Maybe you can just send everything as it is and wait for possible comments about this.